project_template/webpack.config.js

```diff
@@ -14,33 +14,12 @@ const extractOtherCss = new ExtractText("styles.[hash].css");
 
 process.stderr.write(`Building with env = ${env}\n`);
 
-const MAJI_APP = /^MAJI_APP_/i;
-
-const getEnvironmentVariables = function() {
-  return Object.keys(process.env)
-    .filter(key => MAJI_APP.test(key))
-    .reduce((env, key) => {
-      env[`process.env.${key}`] = JSON.stringify(process.env[key]);
-      return env;
-    }, {});
-};
-
-const getGitRevision = function() {
-  const GitRevPlugin = require("git-revision-webpack-plugin");
-  return new GitRevPlugin({
-    commithashCommand: "rev-parse --short HEAD 2> /dev/null || echo untracked"
-  }).commithash();
-};
-
-const getNpmVersion = function() {
-  return require("./package.json").version;
-};
-
 // plugin management
 const HTML = require("html-webpack-plugin");
 const Clean = require("clean-webpack-plugin");
 const SpriteLoaderPlugin = require("svg-sprite-loader/plugin");
 const plugins = [
+  ...require("maji/webpack").plugins,
```

Comment:
If you want this import to work, you'll need to add it at the root of the project currently. Otherwise the import path will be `maji/src/webpack`, which is ugly IMO.